app/jobs/generate_new_rota.rb

```diff
@@ -1,15 +1,30 @@
+require_relative "../../lib/rota_generation"
+
 class GenerateNewRota < Que::Job
```

Comment:
This should extend ActiveJob::Base so we can switch out the actual job runner without needing to change the job implementation